data-prepper-plugins/aws-plugin-api/build.gradle

```diff
@@ -12,7 +14,7 @@ jacocoTestCoverageVerification {
     violationRules {
         rule {
             limit {
-                minimum = 1.0
+                minimum = 0.1
```

Comment:
This project has 100% coverage and can retain that. Dropping this to 10% will prevent us from ensuring that code coverage exists within the project. We should test the `AwsRequestSigningApacheInterceptor`. I believe that a different PR may be doing something similar here.

Reply:
@dlvenable Resolved